src/Esprima/Ast/Literal.cs

```diff
@@ -7,33 +7,26 @@ namespace Esprima.Ast;
 
 public sealed class Literal : Expression
 {
-    private static readonly object _boxedTrue = true;
-    private static readonly object _boxedFalse = false;
+    private static readonly object s_boxedTrue = true;
+    private static readonly object s_boxedFalse = false;
```

Comment:
Now that I remember, I'm not sure if `s_` prefix should be used (editor config might be wrong), Sebastien's other projects don't use it as a suffix

Reply:
I changed this because the IDE suggested it. *.editorconfig* has this currently:
```
dotnet_naming_style.static_field_style.required_prefix = s_
```

Reply:
Yes, the original was taking from.. was it Roslyn repo and might have had some cruft left behind. I think other repos just don't follow the `s_`, but not a concern for this PR.